@xen-orchestra/lite/src/components/form/FormInput.vue

```diff
@@ -75,12 +85,17 @@ const emit = defineEmits<{
 }>();
 
 const value = useVModel(props, "modelValue", emit);
-const empty = computed(() => isEmpty(props.modelValue));
-const isSelect = inject("isSelect", false);
+const empty = computed(
+  () =>
+    props.modelValue == null ||
+    props.modelValue === "" ||
+    props.modelValue === 0
```

Comment:
I asked that myself too 🤔 You're probably right. I decided to include 0 as empty value since it's only for visual purpose.

I also planned to create a custom `isEmpty` function, but it's really hard to define what is empty and what is not.